### PR TITLE
Fix LogMediator property issues

### PR DESCRIPTION
**Pull request by dilee**
## Purpose
Fixing LogMediator property issues.

### CODE REVIEW

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/impl/LogPropertyPropertiesEditionPartImpl.java [hunk 1]
```diff
@@ -507,32 +508,45 @@ protected Composite createPropertyExpressionText(final Composite parent) {
 		propertyExpressionTypeElements = new Control[] { propertyExpressionTextLabel, propertyExpressionText,
 				propertyExpressionHelp };// mouse
 		propertyExpressionText.addMouseListener(new MouseListener() {
+
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/impl/LogPropertyPropertiesEditionPartImpl.java]: Fix formatting issue.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/impl/LogPropertyPropertiesEditionPartImpl.java [hunk 2]
```diff
@@ -507,32 +508,45 @@ protected Composite createPropertyExpressionText(final Composite parent) {
...
-				// TODO Auto-generated method stub
-
-			}
+			public void mouseDoubleClick(MouseEvent e) {}
+			
 		});
+		
+		propertyExpressionText.addKeyListener(new KeyListener() {
+            
+            @Override
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/impl/LogPropertyPropertiesEditionPartImpl.java]: Fix formatting issue.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/impl/LogPropertyPropertiesEditionPartImpl.java (hunk 1 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/impl/LogPropertyPropertiesEditionPartImpl.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/impl/LogPropertyPropertiesEditionPartImpl.java (hunk 2 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/impl/LogPropertyPropertiesEditionPartImpl.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804

**Pull request merged**